### ISSUE TITLE
Snapshot unit test fails on Windows

### ISSUE THREAD

**Issue by ericpromislow** [rancher-sandbox/rancher-desktop#6020]
### Problem Description

`yarn postinstall` doesn't report any errors, but trying rdctl unit tests on Windows fails:

```
PS C:\Users\eric3\rancher-desktop> yarn test:unit:rdctl
yarn run v1.22.21
$ cd ./src/go/rdctl/ && go test ./...
# github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/snapshot [github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/snapshot.test]
pkg\snapshot\manager_windows_test.go:45:22: manager.Snapshotter.WSL undefined (type Snapshotter has no field or method WSL)
...
FAIL    github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/snapshot [build failed]
```

### Proposed Solution

Actually I don't have a solution. The code looks correct, but I might be overlooking some subtlety in go.

**Comment by jandubois** [comment]
This should fix the compilation error, and the failing tests:

```diff
--- src/go/rdctl/pkg/snapshot/manager_windows_test.go
+++ src/go/rdctl/pkg/snapshot/manager_windows_test.go
@@ -37,12 +37,13 @@ func populateFiles(t *testing.T, _ bool) (paths.Paths, map[string]TestFile) {
 }

 func newTestManager(appPaths paths.Paths) *Manager {
+       snapshotter := NewSnapshotterImpl()
+       snapshotter.WSL = wsl.MockWSL{}
        manager := &Manager{
                Paths:         appPaths,
-               Snapshotter:   NewSnapshotterImpl(),
+               Snapshotter:   snapshotter,
                BackendLocker: &lock.MockBackendLock{},
        }
-       manager.Snapshotter.WSL = wsl.MockWSL{}
        return manager
 }

@@ -81,7 +82,7 @@ func TestManagerWindows(t *testing.T) {
                                t.Fatalf("failed to modify %s: %s", testFileName, err)
                        }
                }
-               if err := manager.Restore(snapshot.ID); err != nil {
+               if err := manager.Restore(snapshot.Name); err != nil {
                        t.Fatalf("failed to restore snapshot: %s", err)
                }
                for testFileName, testFile := range testFiles {
@@ -112,7 +113,7 @@ func TestManagerWindows(t *testing.T) {
                                t.Fatalf("failed to remove test directory %q: %s", testDir, err)
                        }
                }
-               if err := manager.Restore(snapshot.ID); err != nil {
+               if err := manager.Restore(snapshot.Name); err != nil {
                        t.Fatalf("failed to restore snapshot: %s", err)
                }
                for _, testDir := range testDirs {
```

> The code looks correct, but I might be overlooking some subtlety in go.

The issue is that `manager.Snapshotter` is a `Snapshotter` interface, but the `WSL` field exist on the `SnapshotterImpl` type.

**Comment by jandubois** [comment]
This has been fixed by #6032 I think.
